### PR TITLE
PP-6008-Add-delete-payment-link-cypress-test

### PR DESCRIPTION
**Pull request by nimalank7**
Description:
- Moves PrototypeProductPaymentIT.shouldDeleteATestProductPayment from selfservice endtoend as a cypress test

### CODE REVIEW

File: test/cypress/integration/payment-links/delete_payment_links_spec.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+const product = buildPaymentLinkOpts('Pay for a parking permit', 'example.com/my-service/pay-for-a-parking-permit', 'en')
+
+describe('Should delete payment link', () => {
+  beforeEach(() => {
+    cy.setEncryptedCookies(userExternalId, gatewayAccountId)
+    cy.task('setupStubs', [
+      commonStubs.getUserStub(userExternalId, [gatewayAccountId]),
+      commonStubs.getGatewayAccountStub(gatewayAccountId, 'test', 'worldpay'),
```
Comment (stephencdaly) [inline test/cypress/integration/payment-links/delete_payment_links_spec.js]: i know this was copy+pasted but could we use destructuring for these imports too for consistency.

File: test/cypress/integration/payment-links/delete_payment_links_spec.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    language: language,
+    type: 'ADHOC',
+    links: [
+      {
+        'rel': 'friendly',
+        'method': 'GET',
+        'href': href
+      }
+    ]
+  }
```
Comment (stephencdaly) [inline test/cypress/integration/payment-links/delete_payment_links_spec.js]: I think we don't have to provide any of these options apart from `external_id`, we can just use the defaults given to us by the fixture hopefully as `external_id` is the only thing used by any of our tests.

File: test/cypress/integration/payment-links/delete_payment_links_spec.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+const product = buildPaymentLinkOpts('Pay for a parking permit', 'example.com/my-service/pay-for-a-parking-permit', 'en')
```
Comment (stephencdaly) [inline test/cypress/integration/payment-links/delete_payment_links_spec.js]: As we only call this function once, I don't think its worthwhile having a function at all, just make it a constant. I.e.
```
const product = { external_id: productExternalId }
```

File: test/cypress/integration/payment-links/delete_payment_links_spec.js (hunk 2 above)
Comment (nimalank7) [reply test/cypress/integration/payment-links/delete_payment_links_spec.js]: I just removed it but it broke the test

**Pull request merged**